src/components/Proposal/PredictionBox.tsx

```diff
@@ -15,6 +15,7 @@ interface IState {
 interface IProps {
   currentPrediction: number;
   currentStake: number;
+  stakerTokens: number;
```

Comment:
I would call this something different like currentAccountTokens - since they are not necessarily a staker yet

Reply:
updated: https://github.com/daostack/alchemy/pull/39/commits/b8dd04d7e8e23e73e18aeadf3395fec6333d802a